buildSrc/src/main/groovy/io.micronaut.build.internal.jms-tests.gradle

```diff
@@ -0,0 +1,21 @@
+plugins {
+    id 'groovy'
+    id 'io.micronaut.build.internal.jms-base'
+    id 'io.micronaut.application'
+}
+
+application {
+    mainClass = 'example.Application'
+}
+
+micronaut {
+    version project.ext.micronautVersion
+    runtime 'netty'
+    testRuntime 'spock'
+}
+
+dependencies {
+    testImplementation "io.micronaut.test:micronaut-test-core:$micronautTestVersion"
```

Comment:
Nothing to do yet but as we move modules to use version catalogs we'll have to support a couple of things:

- versions like `micronautTestVersion` will need to be declared in the catalog, not `gradle.properties`
- the BOM update workflow will need to support updating from the catalog, currently it only supports `gradle.properties`